### PR TITLE
Change from using .cmd to .ps1

### PR DESCRIPTION
**Pull request by NCarlsonMSFT**

### CODE REVIEW

File: generators/app/templates/_dockerTaskGeneric.ps1 [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+            $status = [int]$response.StatusCode
+        }
+        catch [System.Net.WebException] { }
+        if($status -ne 200) {
+            Write-Host "." -NoNewline
+            Start-Sleep 1
+        }
+    }
+
+    Write-Host
```
Comment (PeterJausovec) [inline generators/app/templates/_dockerTaskGeneric.ps1]: empty Write-Host?

File: generators/app/templates/_dockerTaskGeneric.ps1 [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+.PARAMETER Build
+Builds a Docker image.
+.PARAMETER Clean
+Removes the image <%= imageName %> and kills all containers based on that image.
+.PARAMETER Environment
+The enviorment to build for (Debug or Release), defaults to Debug
+.EXAMPLE
+C:\PS> .\dockerTask.ps1 -Build
+Build a Docker image named <%= imageName %>
+#>
```
Comment (PeterJausovec) [inline generators/app/templates/_dockerTaskGeneric.ps1]: Should we show usage? Or is the synposis the equivalent of that?

File: generators/app/templates/_dockerTaskGeneric.ps1 (hunk 1 above)
Comment (NCarlsonMSFT) [inline generators/app/templates/_dockerTaskGeneric.ps1]: This one ends the line Open Site.+
That way if there is an error message or similar afterwards it is on a newline.

File: generators/app/templates/_dockerTaskGeneric.ps1 (hunk 2 above)
Comment (NCarlsonMSFT) [inline generators/app/templates/_dockerTaskGeneric.ps1]: This comment sets up integration with PowerShell's Get-Help feature. Attached is the basic output
[help.txt](https://github.com/Microsoft/generator-docker/files/54331/help.txt)

**Pull request merged**